Reporting-ETL/KK/Reporting_ETL_Job/src/main/java/final_etl/subscribers_0_2/Subscribers.java

```diff
@@ -4298,7 +4298,7 @@ public void tMysqlInput_1Process(
+				String dbquery_tMysqlInput_1 = "select nms_subscribers.id, \n       (CASE WHEN nms_mcts_mothers.name IS NULL THEN nms_mcts_children.name ELSE nms_mcts_mothers.name END) as name,\n	   nms_languages.name as lang,\n       TIMESTAMPDIFF(YEAR, DATE(nms_mcts_mothers.dateOfBirth), CURDATE())AS age,\n       nms_subscribers.dateOfBirth,\n       (CASE WHEN nms_mcts_mothers.state_id_OID IS NULL THEN nms_mcts_children.state_id_OID ELSE nms_mcts_mothers.state_id_OID END) as state,\n       (CASE WHEN nms_mcts_mothers.district_id_OID IS NULL THEN nms_mcts_children.district_id_OID ELSE nms_mcts_mothers.district_id_OID END) as district,\n       (CASE WHEN nms_mcts_mothers.taluka_id_OID IS NULL THEN nms_mcts_children.taluka_id_OID ELSE nms_mcts_mothers.taluka_id_OID END) as taluka,\n       (CASE WHEN nms_mcts_mothers.village_id_OID IS NULL THEN nms_mcts_children.village_id_OID ELSE nms_mcts_mothers.village_id_OID END) as village,\n       (CASE WHEN nms_mcts_mothers.healthBlock_id_OID IS NULL THEN nms_mcts_children.healthBlock_id_OID ELSE nms_mcts_mothers.healthBlock_id_OID END) as hb,\n       (CASE WHEN nms_mcts_mothers.primaryHealthCenter_id_OID IS NULL THEN nms_mcts_children.primaryHealthCenter_id_OID ELSE nms_mcts_mothers.primaryHealthCenter_id_OID END) as hf,\n       (CASE WHEN nms_mcts_mothers.healthSubFacility_id_OID IS NULL THEN nms_mcts_children.healthSubFacility_id_OID ELSE nms_mcts_mothers.healthSubFacility_id_OID END) as hsf,\nnow() as lastModifiedTime,      \nnms_subscribers.lastMenstrualPeriod,GREATEST(nms_subscribers.modificationDate,COALESCE(nms_mcts_mothers.modificationDate,0),\n	  COALESCE(nms_mcts_children.modificationDate,0),COALESCE(nms_languages.modificationDate,0)) as 'modificationDate'\nFROM nms_subscribers\nLEFT OUTER JOIN nms_mcts_mothers ON nms_subscribers.mother_id_OID = nms_mcts_mothers.id\nLEFT OUTER JOIN nms_mcts_children ON nms_subscribers.child_id_OID = nms_mcts_children.id\nLEFT OUTER JOIN nms_languages ON nms_subscribers.language_id_OID = nms_languages.id\nWHERE nms_subscribers.creationDate>'"
```

Comment:
the only change here is that the mapping is now done with primaryHealthCenter_id_OID instead of healthFacility_id_OID, right?